sklearn/utils/estimator_checks.py

```diff
@@ -1323,6 +1327,32 @@ def check_estimators_partial_fit_n_features(name, estimator_orig):
         estimator.partial_fit(X[:, :-1], y)
 
 
+@ignore_warnings(category=(DeprecationWarning, FutureWarning))
+def check_classifier_multioutput(name, estimator_orig):
+
+    estimator_orig = clone(estimator_orig)
+    X, y = make_multilabel_classification(random_state=42)
+    if name not in ['OneVsRestClassifier', 'RidgeClassifierCV']:
+        estimator_orig.fit(X, y.astype(str)).predict(X)
+    estimator_orig.fit(X, y).predict(X)
+
+
+@ignore_warnings(category=(DeprecationWarning, FutureWarning))
+def check_regressor_multioutput(name, estimator_orig):
+
+    estimator_orig = clone(estimator_orig)
+    X, y = make_regression(random_state=42, n_samples=100, n_targets=5)
+
+    tags = _safe_tags(estimator_orig)
+    if tags["requires_positive_X"]:
+        X = np.abs(X)
+
+    if name == "RANSACRegressor":
+        estimator_orig.min_samples = 99
+
+    estimator_orig.fit(X, y).predict(X)
```

Comment:
We should check the shape of the output.

Reply:
Added checks, please review.